src/dtos/OrderItemDto.ts

```diff
@@ -0,0 +1,7 @@
+export interface OrderItemDto {
+  id: number;
+  name: string;
+  price: number;

```

Comment:
We don't want these files added do we?

Reply:
And why has it rebuilt all the files?

Reply:
Oh oops didn't even notice the files being added back - removed them. The line endings of all files were changed from LF to CRLF which is why they show up as affected

Reply:
Ah yes, I see! Ok will approve